### DISCUSSION TITLE
目前Python、java、PHP、Go、Rust这几个语言的SDK哪个最成熟？

### DISCUSSION THREAD

**Discussion by jmjoy** [wechaty/wechaty#2562] [Q&A]
我不会Nodejs和typescript，只会这几个语言，想问下哪个最成熟。

我这边需要的功能是：接收、发送聊天记录，创建解散群聊，获取好友列表和信息等。

**Comment by zhihuifanqiechaodan** [comment]
> 我不会Nodejs和typescript，只会这几个语言，想问下哪个最成熟。
> 
> 
> 
> 我这边需要的功能是：接收、发送聊天记录，创建解散群聊，获取好友列表和信息等。

最成熟的是node一直官方在维护，其余的可能少功能，需要的话我这边可以写
